Sources/NavigationBackport/NBNavigationStack.swift

```diff
@@ -12,7 +14,7 @@ class PathHolder<Data>: ObservableObject {
 @available(iOS, deprecated: 16.0, message: "Use SwiftUI's Navigation API beyond iOS 15")
 public struct NBNavigationStack<Root: View, Data: Hashable>: View {
   @Binding var path: [Data]
-  @ObservedObject var pathHolder: PathHolder<Data>
+  @Backport.StateObject var pathHolder: PathHolder<Data>
```

Comment:
Is this line change necessary? It was previously an `ObservedObject`.

Reply:
Whups, I meant to ask about this!

My impression was that `@ObservedObject` is non-owning; before I noticed that there was a `@StateObject` elsewhere, my first theory to why the crash was happening was that maybe this was getting released which was causing some UB or something somewhere...

So, why _does_ it work the way it was before? I'd expect the `PathHolder` to get released!

Reply:
Thanks, you're right - I don't think this should ever have been an `@ObservedObject`. I think I can simplify it to just use `@State` instead: https://github.com/johnpatrickmorgan/NavigationBackport/compare/feature/state-for-owned-path